app/models/miq_worker.rb

```diff
@@ -329,15 +329,23 @@ def send_message_to_worker_monitor(message, *args)
   end
 
   def start
-    msg = "Worker started: ID [#{self.id}], PID [#{self.pid}], GUID [#{self.guid}]"
-    MiqEvent.raise_evm_event_queue(self.miq_server, "evm_worker_start", :event_details => msg, :type => self.class.name)
+    pid = fork do
+      ActiveRecord::Base.connection.reconnect!
+      DRb.stop_service
```

Comment:
Need to deal with the established DB connection and DRb service in the parent process since all forked workers inherit them.